masterlist.yaml

```
@@ -9884,6 +9884,31 @@ plugins:
         util: '[TES4Edit v4.0.3j (RC2 for 4.0.4)](https://www.nexusmods.com/oblivion/mods/11536)'
         itm: 9
 
+  - name: 'sunk ships.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/13467/'
+        name: 'Ship Wrecks'
+    msg:
+      - <<: *wildEdits
+        subs: [ 'Remove **00033569**, **000CA158**, **00015C3D**, & **00006622** from **sunk ships.esp**.' ]
+        condition: 'checksum("sunk ships.esp", E238F1EC) or checksum("sunk ships.esp", 3D7DA867)'
+    tag:
+      - Actors.RecordFlags
+      - C.Music
+      - C.Name
+      - C.Water
+    dirty:
+      - <<: *quickClean
+        crc: 0xE238F1EC
+        util: '[TES4Edit v4.0.4b](https://www.nexusmods.com/oblivion/mods/11536)'
+        itm: 131
+        udr: 34
+      - <<: *quickClean
+        crc: 0xFC7501E3
+        util: '[TES4Edit v4.0.4b](https://www.nexusmods.com/oblivion/mods/11536)'
+        itm: 130
+        udr: 34
```

Sorry, forgot to consider that not deleting that last record would affect the cleaning data.

```suggestion
      - <<: *quickClean
        crc: 0x9C91D9B1
        util: '[TES4Edit v4.0.4b](https://www.nexusmods.com/oblivion/mods/11536)'
        itm: 131
        udr: 34
```